docs/validator/economics.md

```diff
@@ -0,0 +1,136 @@
+</blockquote>
+
+
+## 2. Nodes infrastructure
+
+Once you obtained your stake in $NEAR, you need to setup the infrastructure to serve the network.
+At a high level, your nodes must provide:
+- Availability
+- Consistency
+
+### Availability
+
+NEAR Protocol doesn't enforce slashing for downtime. However, validators failing to be online get *kicked out* and lose rewards.
+More in detail, the protocol requires your nodes to provide a `MINIMUM_THRESHOLD` of signatures, in a number proportional to the median block producer in the `epoch`, multiplied by the number of *seats* assigned to your node. If your node uptime falls below `90 %`, it will be *kicked out* at the end of the epoch, and you will have to submit its staking `proposal` again.
+In practical terms, if your node goes offline for too long, you have to:
+- Re-stake your `proposal` again before the end of the failed epoch, and (if your `proposal` succeed),
+- Wait for one more `epoch`, which is needed to sync the node with the assigned shard
+
+With 730 epochs per year (one every 12 hours), you will miss at least 24 hours of potential rewards.
+
+### Consistency
+
+Validators must submit a *valid state* to the blockchain, or their stake will get slashed. Beware that validators may be slashed if any fallback systems intervene by mistake, generating blocks with the same key across different nodes. Therefore, as a validator, you need a reliable solution that maintains consistency across different datacenters and geographies.
+There are many more slashing conditions, with documentation that is work in progress, please refer to the [EpochManager specs](https://github.com/nearprotocol/NEPs/pull/37) if you want to know more.
+
+<blockquote class="warning">
+    <strong>heads up</strong><br><br>
+    At the launch of MainNet, NEAR will have no slashing. Therefore, consistency requirements will be initially lower
+</blockquote>
+
+
+## 3. Continuous operation
+
+Validators must keep the operations up and running by:
+1. Setting the *Service Level Objectives*
+2. Understanding market dynamics
+
+### Setting the Service Level Objectives
+
+As mentioned above, there's no slashing for downtime. However, any new release of the node should be locally tested and deployed with minimum interruptions, to avoid the risks of getting *kicked out*. Allocating resources on infrastructure-as-code can increase efficiency and profitability, making ongoing maintenance easier.
+On top of ordinary maintenance, your operations will require an incident/response playbook, to promptly resolve any infrastructure failure or emergency updates.
+
+
+### Understand market dynamics
+
+NEAR validators are subject to market and game-theory dynamics, relating to:
+1. The total of NEAR tokens at stake
+2. The distribution of the stake across validators
+3. Protocol fees and inflation
```

Comment:
what does "protocol fees" mean? Does this refer to transaction fees?

Reply:
corrected. Yes it's transaction fees - I agree it's more clear